library/swift/resources/BUILD

```diff
@@ -0,0 +1,6 @@
+licenses(["notice"])
+
+exports_files(
```

Comment:
if you just don't have a BUILD file in this dir you can reference this file directly. you just can't do that across package boundaries